examples/Cassie/cassie_test.pmd

```diff
@@ -14,47 +14,52 @@ group "1.simulated-robot" {
         exec = "bazel-bin/examples/Cassie/run_pd_controller";
         host = "localhost";
     }
-    cmd "2.a.dispatcher-robot-out (lcm)" {
```

Comment:
We should probably take a moment to organize these procman configs because they are getting fairly bloated - maybe after the lab meeting?

Reply:
Yea I think we should separate the ones for simulation from those for the hardware. We can talk in Slack to find a time to discuss about this